webapp/channels/src/utils/constants.tsx

```diff
@@ -2029,6 +2036,65 @@ export const WindowSizes = {
     DESKTOP_VIEW: 'desktopView',
 };
 
+export enum SidebarSize {
```

Comment:
@kyeongsoosoo Could you please move these to separate file.

Reply:
I moved these to the sidebar component folder.